src/components/Row/Row.tsx

```diff
@@ -0,0 +1,43 @@
+import { formatAmount, formatDate } from '../../utils/formatters';
+import Chip from '../Chip/Chip';
+import Contact from '../Contact/Contact';
+
+interface RowProps {
+  status: 'paid' | 'partial' | 'unpaid';
+  createdAt: Date;
+  contact: LocalContact;
+  amount: number;
+  currency: 'EUR' | 'GBP';
+  tags: string[];
+}
+
+const Row = ({ status, createdAt, contact, amount, currency, tags }: RowProps) => {
```

Comment:
Just thinking. Would it not make more sense to call this PaymentRequestRow and pass a PaymentRequest into it? In future the PaymentRequest Details will be triggered here? I don't think it makes sense to create a resuable row component at this level. Other views will probably have completely different fields/details.

Reply:
Fair enough. We should probably look at trimming down our ApiRsesponses as well. I created them as dupes of the api versions for starters.

Reply:
Yep. That's why I started creating "Local" types (i.e `LocalCurrency`), following the advice from our [bible](https://martinfowler.com/articles/modularizing-react-apps.html).

In a near future we'd find ourselves mapping the API types to the Local ones.

---

Btw, the renaming is already up🤘🏻